salesforce/management/commands/update_opportunities.py

```diff
@@ -11,30 +12,46 @@ def handle(self, *args, **options):
             now = datetime.datetime.now()
 
             year = now.year
-            if now.month < 7: # Salesforce needs the school base year, this is how they calculate it
+            if now.month > 7:  # Salesforce needs the school base year, this is how they calculate it
```

Comment:
@dmoreno27 can you sanity check how i'm calculating this date and the query below?

Reply:
it seems like it might have grabbed the wrong base year before... not sure how it ever worked

If it's _after_ August, it should get the base year _before_ the current year.